packages/client/src/pages/PortalSettings/categories/integration/SingleSignOn/sub-components/AddSpCertificateModal.js

```diff
@@ -64,6 +65,7 @@ const AddSpCertificateModal = (props) => {
         </Text>
 
         <TextArea
+          id="spPrivateKey"
```

Comment:
Rename "spPrivateKey" to "sp-privateKey"

Reply:
Fix to "sp-privateKey"